.github/workflows/system-tests.yml

```diff
@@ -0,0 +1,51 @@
+name: System Tests
+
+on:
+  pull_request:
+    branches:
+      - "**"
+  workflow_dispatch: {}
+  schedule:
+    - cron:  '00 04 * * 2-6'
+
+jobs:
+  system-tests:
+    if: ${{ github.event.pull_request.head.repo.full_name == 'DataDog/dd-trace-py' }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - weblog-variant: flask-poc
+          - weblog-variant: uwsgi-poc
+      fail-fast: false
+    env:
+      TEST_LIBRARY: python
+      WEBLOG_VARIANT: ${{ matrix.weblog-variant }}
+      DD_API_KEY: ${{ secrets.DD_API_KEY }}
+    steps:
+      - name: Checkout system tests
+        uses: actions/checkout@v2
+        with:
+          repository: 'DataDog/system-tests'
+
+      - name: Checkout dd-trace-py
+        uses: actions/checkout@v2
+        with:
+          path: 'binaries/dd-trace-py'
+
+      - name: Build
+        run: ./build.sh
+
+      - name: Run
+        run: ./run.sh
+
+      - name: Compress artifact
+        if: ${{ always() }}
+        run: tar -czvf artifact.tar.gz $(ls | grep logs)
+
+      - name: Upload artifact
+        uses: actions/upload-artifact@v2
+        if: ${{ always() }}
+        with:
+          name: logs_${{ matrix.weblog-variant }}
+          path: artifact.tar.gz
```

Comment:
Can we leave a comment on these two sections that we want to compress and upload the artifacts even on failures.

It is semi-obvious with `always()`, but having a comment just to be sure someone doesn't manually edit and change the expected behavior.